### ISSUE TITLE
Kotlin Bootcamp issue: 

### ISSUE THREAD

**Issue by franc-ist** [google-developer-training/kotlin-bootcamp#63]
**Describe the problem**
Attempting to add `inline` to my code in the place specified:
```kotlin
inline myWith(fish.name) {
    capitalize()
}
```
produces the following error:

`inline` is unresolved reference.

![](https://i.imgur.com/3DCPu6m.png)

**In which step of the codelab can this issue be found?**
Section 6 `Concept: Inline functions` of Kotlin Bootcamp for Programmers 6: Functional manipulation

**Versions**
What version of IntelliJ IDEA are you using?
```
IntelliJ IDEA 2020.1.1 (Ultimate Edition)
Build #IU-201.7223.91, built on April 30, 2020
Runtime version: 11.0.6+8-b765.40 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Kotlin version 1.3.72 (JRE 14.0.1+7)
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 1963M
Cores: 8
Registry: ide.balloon.shadow.size=0
```

**Additional information**
I believe moving `inline` to the function definition for `myWith` produces the desired result.

**codelab:** Kotlin Bootcamp for Programmers 6: Functional manipulation

**Comment by Nickrader** [comment]
Yeah, looks like the instructions 'inline' the call not the definition.  
I agree, 'inline' should be moved to definition.
`inline fun myWith(name: String, block: String.() -> Unit) {
    name.block()
}`
